packaging/nnstreamer.spec

```
@@ -179,7 +179,7 @@ enable_tf=true
 enable_tf=false
```

```suggestion
%define enable_tf false
```
and same for `enable_tf=true`

---

Thanks!